pandas/core/indexes/base.py

```diff
@@ -3151,8 +3151,8 @@ def get_indexer(
                 )
 
             indexer = self._engine.get_indexer(target._get_engine_target())
-
-        return ensure_platform_int(indexer)
+        x = ensure_platform_int(indexer)
```

Comment:
This variable adds nothing but noise to the code

Reply:
Yep this was accidential